ifrs17/Import/Importers.ipynb

```diff
@@ -444,28 +467,19 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task<ImportArgs[]> GetTargetArgsAsync(ImportArgs mainArgs)",
+                "public async Task<ImportArgs[]> GettargetArgsListAsync(ImportArgs mainArgs)",
```

Comment:
GettargetArgsListAsync --> GetTargetArgsAsync
Changes: capital T, remove List (what's the added value? also the return type is not a list)

Reply:
renamed